packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1260,6 +1277,11 @@ export class GarbageCollector implements IGarbageCollector {
                 return;
             }
 
+            const age = currentReferenceTimestampMs - nodeStateTracker.unreferencedTimestampMs;
+            idsToSweep.add({ nodeId, nodeType, age });
+        });
+
+        idsToSweep.forEach(({ nodeId, nodeType, age }) => {
```

Comment:
Do we need this code here?

Reply:
I think continuing to log when we delete stuff will be useful.  I'll probably refactor this back into the upper forEach loop to make it simpler.

Reply:
I specifically meant those lines, of code 1281 - 1284, if you removed those lines, wouldn't the code still work the same?